bots/pr/src/main/java/org/openjdk/skara/bots/pr/LabelerWorkItem.java

```diff
@@ -116,13 +116,13 @@ private void updateLabelMessage(List<Comment> comments, List<String> newLabels)
 
     @Override
     public Collection<WorkItem> prRun(ScratchArea scratchArea) {
-        if (bot.isAutoLabelled(pr)) {
+        if (bot.labelConfiguration().allowed().isEmpty()) {
+            bot.setAutoLabelled(pr);
             return List.of();
         }
 
-        if (bot.labelConfiguration().allowed().isEmpty()) {
-            bot.setAutoLabelled(pr);
-            return List.of(CheckWorkItem.fromWorkItemWithForceUpdate(bot, prId, errorHandler, triggerUpdatedAt));
+        if (bot.isAutoLabelled(pr) || pr.isClosed()) {
+            return List.of();
```

Comment:
Having this check is good, but I think we should also check `!pr.isClosed()` before even creating a `LablerWorkItem` in `PullRequestCommandWorkItem`. That should help alleviate even more startup congestion.